src/crypto/PrivateKey.ts

```diff
@@ -54,7 +54,7 @@ export default class PrivateKey implements proto.PrivateKey {
   }
 
   // sign provided digest
-  async sign(digest: Uint8Array): Promise<Signature> {
+  async sign(digest: Uint8Array | string): Promise<Signature> {
```

Comment:
Do we really need this? I'd rather not have that. If we do then we should be clear on how is the string interpreted (i.e. converted to bytes).

Reply:
Thats fair. Will remove